lib/python/compliance_checker_test.py

```diff
@@ -157,20 +157,16 @@ def run_test_all_collection(compliance_config):
         print("Running test suite for: {collection}".format(collection=collection))
 
         for sub_collection in compliance_config[collection].items():
-            try:
```

Comment:
why removing the try except?

Reply:
Because it has no effect. The only thing it was doing with the caught exception is raising it. That's equivalent to not catching it in the first place, isn't it?